### ISSUE TITLE
It just deleted everything

### ISSUE THREAD

**Issue by Ishraque89** [greguly/WooCommerce-Remove-All-Products#12]
Bruh, even my site logo is gone after installing this!

I just wanted to move the products to trash, and it deleted everything from the wordpress database with no backup!!!
WHY!!!

**Comment by greguly** [comment]
Oh dear, sorry to hear about it, but plugin did exactly what it says on the tin.

Hope you did not run it at a live site.

Site logo being removed is something that should not happen, unless it was also a product image.

**Comment by greguly** [comment]
Reopened in case site logo was not a product image.

**Comment by Ishraque89** [comment]
I think it was included in a draft product, but there should be a **warning** or something.
I had trouble figuring out the product csv file, so I installed a few plugins, most of them were no use, but this one was a nuisance. Fortunately, I came across an auto backup that which might not be available to many people.
Is there a chance to add a **warning that is more transparent** in the future? Thank you.
